data.sql

```diff
@@ -61,4 +61,34 @@ FROM owners
 WHERE owners.full_name='Dean Winchester' AND animals.name='Angemon' 
 OR owners.full_name='Dean Winchester' AND animals.name='Boarmon';
 
+/* 4th PULL REQUEST */
+INSERT INTO vets(name, age, date_of_graduation) VALUES ('William Tatcher', 45, '2000-04-23');
+INSERT INTO vets(name, age, date_of_graduation) VALUES ('Maisy Smith', 26, '2019-01-17');
+INSERT INTO vets(name, age, date_of_graduation) VALUES ('Stephanie Mendez', 64, '1981-05-04');
+INSERT INTO vets(name, age, date_of_graduation) VALUES ('Jack Harkness', 38, '2008-06-08');
 
+INSERT INTO specializations(id_vet, id_species) VALUES (1, 1);
+INSERT INTO specializations(id_vet, id_species) VALUES (3, 1);
+INSERT INTO specializations(id_vet, id_species) VALUES (3, 2);
+INSERT INTO specializations(id_vet, id_species) VALUES (4, 2);
+
+INSERT INTO visits(id_animals, id_vet, visit_date) VALUES(1,1, '2020-05-24');
+INSERT INTO visits(id_animals, id_vet, visit_date) VALUES(1,3, '2020-07-22');
+INSERT INTO visits(id_animals, id_vet, visit_date) VALUES(2,4, '2021-02-02');
+INSERT INTO visits(id_animals, id_vet, visit_date) VALUES(3,2, '2020-01-02');
+INSERT INTO visits(id_animals, id_vet, visit_date) VALUES(3,2, '2020-03-08');
+INSERT INTO visits(id_animals, id_vet, visit_date) VALUES(3,2, '2020-05-14');
+INSERT INTO visits(id_animals, id_vet, visit_date) VALUES(4,3, '2021-05-04');
+INSERT INTO visits(id_animals, id_vet, visit_date) VALUES(5,4, '2021-02-24');
+INSERT INTO visits(id_animals, id_vet, visit_date) VALUES(6,2, '2019-12-21');
+INSERT INTO visits(id_animals, id_vet, visit_date) VALUES(6,1, '2020-08-10');
+INSERT INTO visits(id_animals, id_vet, visit_date) VALUES(6,2, '2021-04-07');
+INSERT INTO visits(id_animals, id_vet, visit_date) VALUES(7,3, '2019-09-29');
+INSERT INTO visits(id_animals, id_vet, visit_date) VALUES(8,4, '2020-10-03');
+INSERT INTO visits(id_animals, id_vet, visit_date) VALUES(8,4, '2020-11-04');
+INSERT INTO visits(id_animals, id_vet, visit_date) VALUES(9,2, '2019-01-24');
+INSERT INTO visits(id_animals, id_vet, visit_date) VALUES(9,2, '2019-05-15');
+INSERT INTO visits(id_animals, id_vet, visit_date) VALUES(9,2, '2019-02-27');
```

Comment:
- [ ] On line 91 the value for visit_date is `2020-02-27` according to the requirements, kindly make the changes accordingly.